src/hazelcore/UHExp.re

```diff
@@ -19,10 +19,12 @@ and operand =
   | IntLit(ErrStatus.t, string)
   | FloatLit(ErrStatus.t, string)
   | BoolLit(ErrStatus.t, bool)
+  | StringLit(ErrStatus.t, string)
   | ListNil(ErrStatus.t)
   | Fun(ErrStatus.t, UHPat.t, t)
   | Inj(ErrStatus.t, InjSide.t, t)
   | Case(CaseErrStatus.t, t, rules)
+  | Subscript(ErrStatus.t, t, t, t)
```

Comment:
It might make more sense to have the subscript target be of type operand rather than t, since we don't allow it to be an opseq or any other kind of block expression anyway.